keyboards/maxr1998/pulse4k/keymaps/default/keymap.c

```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
```suggestion
```

Probably not needed, I don't see it being referenced anywhere.

Reply:
Removed, thanks!